### PR TITLE
yammer.com private social network

### PR DESCRIPTION
**Pull request by TraderStf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
   * https://kato.im - Team Chat & Collaboration, free for unlimited users with some feature limitations
   * http://seafile.com/ - Private or cloud storage, file sharing, sync, discussions. Private version is full. Cloud version has just 1 GB.
   * https://sameroom.io - Free for unlimited users with some feature limitations
+  * https://yammer.com/ - Private social network standalone or for M$ Office 365. Free, just a bit less admin tools and users management features.
```
Comment (ripienaar) [inline README.md]: I cant see what restrictions are on the free version on their site, do you know? Please don't write M$.

**Comment by TraderStf** [comment]
https://about.yammer.com/pricing/
FAQ - UPGRADING - 3rd

Yammer Enterprise networks include admin tools like user management, the ability to authorize and manage apps for your network and access to customer success tools and the Microsoft Help Desk support. Compare detailed features here:

https://about.yammer.com/assets/WhyYammerEnteprise.pdf

**Comment by ripienaar** [comment]
Yeah but that does not say what limitations are for the free edition? I don't see what the free one gives you.

---
R.I.Pienaar

**Comment by TraderStf** [comment]
Yammer Basic = free :-)

What is the difference between Yammer Enterprise (paid subscription) and Yammer Basic (free)? 

Yammer Enterprise networks include admin tools like user management, the ability to authorize and manage apps for your network and access to customer success tools and the Microsoft Help Desk support. Compare detailed features here.

**Pull request merged**